eng/pipelines/mgmt.yml

```diff
@@ -1,25 +1,163 @@
-# DO NOT EDIT THIS FILE
-# This file is generated automatically and any changes will be lost.
-
-trigger: none
-pr: none
-
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - sdk/appconfiguration/Azure.Data.AppConfiguration
```

Comment:
As we chatted given this Pipeline is for mgmt libraries we should do an include for those instead of an exclude for client. That is how all other pipelines in this repo function.

Reply:
I updated the script to include management directories instead.